packages/react-combobox/src/components/ComboButton/ComboButton.test.tsx

```diff
@@ -7,6 +7,7 @@ describe('ComboButton', () => {
   isConformant({
     Component: ComboButton,
```

Comment:
I like `ComboboxButton` 👍

Part of the difficulty with `Option` is that it is also the child for `<Listbox>`, which can be used as a standalone component.

I've made an issue to iron out naming for the package in general, since it looks like there'll also be Dropdown vs. Combobox, where the only difference is the trigger component: #21989. It's also added to the component implementation checklist: https://github.com/microsoft/fluentui/issues/18133